### DISCUSSION TITLE
Write several variables with only one command

### DISCUSSION THREAD

**Discussion by BusinessClaes** [arendst/Tasmota#16827] [Q&A] [answered]
Hi!

Does someone know if it is possible to write several variables with only one command, instead of one command for each variable? I can’t find any on this in the docs.

I’ve tried all the combinations I can comeup with, but none of them does the trick
`var 10,20,30`
`var {10,20,30}`
`var=10,20,30`
`var[1,2,3]={10,20,30}`
none of these work…

**Comment by sfromis** [comment, marked as answer]
No such feature is implemented. Variables in rules can not be accessed like sets or lists, but simply as independent slots. If such was included, it would be mentioned in the docs too.

Anyway, here is one command modifying multible variables...
`Backlog var1 10; var2 20; var3 30`
Yes, Backlog is one command, able to house multiple actions in the form of other commands.

**Reply by BusinessClaes** [reply]
Yes, that’s the way I am doing it now. I just wondered if I was missing a neater way. Thanks for answering!
